### DISCUSSION TITLE
error installing - I am getting error when trying to install dex

### DISCUSSION THREAD

**Discussion by YupaTT** [dexidp/dex#3364] [Q&A]
google.golang.org/grpc/reflection/grpc_reflection_v1alpha
google.golang.org/grpc/reflection
github.com/dexidp/dex/cmd/dex
# github.com/dexidp/dex/cmd/dex
cmd/dex/config.go:183:26: undefined: sql.SQLite3
cmd/dex/config.go:206:43: undefined: sql.SQLite3
make: *** [Makefile:45: bin/dex] Error 1

**Comment by nabokihms** [comment]
@YupaTT could you please provide more info about your installation? 
* Do you use official helm chart? 
* Do you use official docker image?
* Which version?
* Architecture of your server, e.g., amd64, arm

**Reply by YupaTT** [reply]
Perform the clean installation with golang 1.9 and 1.22 as indicated on the official page on Ubuntu server and on Debian with amd64 architecture following these steps:

```
$git clone https://github.com/dexidp/dex.git
$ cd dex/
$make build
```

and with both computers it presents this problem.
